pkg/cmd/pulumi/refresh.go

```diff
@@ -140,6 +150,55 @@ func newRefreshCmd() *cobra.Command {
 				return result.FromError(fmt.Errorf("getting stack configuration: %w", err))
 			}
 
+			if skipPendingCreates && clearPendingCreates {
```

Comment:
Same check for skipPending and importPending?

Reply:
`--skip-pending-creates` implies don't show me existing pending creates.
`--clear-pending-creates` implies remove pending creates.
It doesn't make sense to provide a command that ensures there will be no pending creates *and* that you shouldn't show pending creates (since there will never be any).

`--import-pending-creates` turns some set of pending creates into imports (finishing the create). 
It makes sense to run `--import-pending-creates` & `--skip-pending-creates`, telling the CLI to import a set of resources and not prompt for the rest.

It even makes sense to run `--import-pending-creates` & `--clear-pending-creates`, telling the CLI to import a set of resources and remove the rest.

> Same check for skipPending and importPending?

I think it is valid to combine `--skip-pending-creates` with `--import-pending-creates`.